go.mod

```diff
@@ -15,13 +15,27 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
```

Comment:
can you run `go mod tidy`? it's weird that deps have been managed this way 🤔

Reply:
Done without any effect, it's also being done by the `make lint`.
I'll try inside devcontainer

Reply:
recreated file